src/elements/kw-social-comment/kw-social-comment.html

```diff
@@ -0,0 +1,194 @@
+<link rel="import" href="../../bower_components/polymer/polymer.html">
+<link rel="import" href="../../bower_components/iron-flex-layout/iron-flex-layout.html">
+<link rel="import" href="../../bower_components/iron-image/iron-image.html">
+<link rel="import" href="../../bower_components/iron-list/iron-list.html">
+<link rel="import" href="../../bower_components/iron-scroll-threshold/iron-scroll-threshold.html">
+<link rel="import" href="../kano-api-client/kano-api-client-behavior.html">
+<link rel="import" href="../../bower_components/date-util/date-util.html">
+<link rel="import" href="../kano-style/color.html">
+<dom-module id="kw-social-comment">
+    <template>
+        <style>
+             :host {
+                display: block;
```

Comment:
i would remove 'display block' as that gets overwritten with the @apply layout for 'display: flex', for clarity